.travis.yml

```diff
@@ -25,4 +25,5 @@ after_success:
   - if [ -z "$DOCKER_PASS" ]; then echo "Build triggered by external PR. Skipping docker push" && exit 0; fi
   - docker login -u $DOCKER_USER -p $DOCKER_PASS;
   - ./scripts/build.sh
+  - ./scripts/smokey.sh
```

Comment:
This is a container level test. Not a smoke test. Rename to `container-test.sh` to be consistent with other repos.